pyccel/ast/basic.py

```diff
@@ -66,7 +66,6 @@ def __init__(self):
         self._recursion_in_progress = False
         for c_name in self._my_attribute_nodes: #pylint: disable=not-an-iterable
             c = getattr(self, c_name)
-
```

Comment:
This change does not seem relevant to this PR. Please try to remove irrelevant changes from PRs so that they are as small as possible. This makes them easier to review (this will be especially important in the future when your PRs are larger)